tools/mcsema_disass/ida/get_cfg.py

```diff
@@ -1040,11 +1040,74 @@ def recover_regions(M, exported_vars, global_vars=[]):
     seg_name = idc.SegName(seg_ea)
     seg_names[seg_ea] = seg_name
 
-    if not is_external_segment_by_flags(seg_ea) or \
-       segment_contains_external_function_pointers(seg_ea):
+    if (not is_external_segment_by_flags(seg_ea) or \
+        segment_contains_external_function_pointers(seg_ea)) and \
+        not (is_constructor_segment(seg_ea) or is_destructor_segment(seg_ea)):
       seg_parts[seg_ea].add(seg_ea)
       seg_parts[seg_ea].add(idc.SegEnd(seg_ea))
 
+    # Issue #330
+    # Fix for an imporatnt feature - static storage allocation of the objects in C++, where
```

Comment:
"imporatnt" -> "important"

If this fixes #330 completely then I would remove the reference here (line 1049).